lib/muffin_man/notifications/v1.rb

```diff
@@ -0,0 +1,63 @@
+module MuffinMan
+  module Notifications
+    require "json"
+    require "sp_api_helpers"
+    class V1 < SpApiClient
+
+      NOTIFICATION_PATH = "/notifications/v1"
+
+      def create_destinations(arn, params={})
+        params = params.transform_keys{|key| key.to_s}
+        @local_var_path = "#{NOTIFICATION_PATH}/destinations"
+        destination_params = {"resourceSpecification": {"sqs": {"arn": arn}}, "marketplaceIds": params["marketplace_ids"]}
+        destination_params.merge!("eventBridge": {"region": params["region"], "accountId": params["account_id"]}) unless params["region"].nil? || params["account_id"].nil?
+        destination_params.merge!("name": params["name"]) unless params["name"].nil?
```

Comment:
Can we make `name` a required input to this method since "name" is a required parameter?